### ISSUE TITLE
Invisible Trapdoors using recommended forge version (1.12.2)

### ISSUE THREAD

**Issue by ate0ate** [Ordinastie/MalisisDoors#247]
Forge: 14.23.2.2654
Malisis Core: 6.4.0
Malisis Doors 7.3.0
Malisis Blocks 6.1.0

In a custom modpack, closest to All the Mods 3 I suppose, iron trapdoors are invisible. Opened or closed, it doesn't matter. Other trapdoors seem fine and the issue goes away when removing the Malisis mods. I am using Optifine if that makes a difference, but I'd imagine not as the issue is isolated to iron trapdoors. Thank you for your time. Edit: Yeah, removing Optifine does nothing.

**Comment by Ayasu** [comment]
Came here to check about posting of this bug and confirm the same is happening to all vanilla minecraft trapdoors and Malisis trapdoors.

Considering ate0ate and I are on the same version of Forge, it looks like it might be an issue with more recent Forge builds and this mod.

**Comment by ate0ate** [comment]
Ayasu is correct I believe. I had thought the other trapdoors worked fine as I had placed them prior to updating forge. Only after had I noticed the iron trapdoor being invisible. They are indeed all invisible upon checking again.

**Comment by ate0ate** [comment]
Okay, after a lot of testing mods one at a time with Malisis Doors I narrowed the issue down to Secret Rooms. Just Forge, Secret Rooms, and Malisis Core/Doors is all that is needed to trigger the issue of trapdoors being invisible. Not sure which mod is actually the cause.

**Comment by Ayasu** [comment]
Newest Forge(at the time) + Malisisdoors was doing it for me. I wouldn't be shocked if Secret Rooms was being wonky, but before updating Forge last, everything was working just fine. Secret Rooms updated more  recently than my reply to this thread, so hopefully maybe? We'll see.

**Comment by ate0ate** [comment]
@Ayasu Check out issue #250. They got a little miffed when I mentioned it was forge and they said no, it worked fine in their tests. I checked the Malisis Doors/Core alone with the recommended forge (2655) and it worked fine. Added just Secret Rooms with it and all the trap doors went invisible. The person who posted that issue is also having rendering problems and is not using Secret Rooms, at least according to their mod list. Looks like there are other mod interactions out there but forge and Malisis do indeed work all on their own. Idk. Confusing.
